src/main/java/util/Ui.java

```diff
@@ -0,0 +1,146 @@
+package util;
+
+import task.Task;
+
+import java.util.Scanner;
+
+/**
+ * The UI class deals with interactions with the user and prints responses accordingly.
+ */
+public class Ui {
+    /**
+     * Scanner to read in user input input.
+     */
+    private final Scanner sc;
+
+    /**
+     * Creates a UI instance and create a new scanner object.
+     */
+    public Ui() {
+        sc = new Scanner(System.in);
+    }
+
+    /**
+     * Returns a string describing a command.
+     *
+     * @return A string describing a command.
+     */
+    public String readCommand() {
+        return sc.nextLine();
+    }
+
+    /**
+     * Prints a line divider.
+     */
+    public void showLine() {
+        System.out.println("---------------------------------------------------------------------------------------" +
+                "-----------------------------------");
+    }
+
+    /**
+     * Prints the welcome message.
+     */
+    public void showWelcome() {
+        System.out.println("Serina here, how can I assist you? Enter 'help' to learn what I can do for you.");
+        this.showLine();
+    }
+
+    /**
+     * Prints the help message with instructions on how to use Serina.
+     */
+    public void showHelp() {
```

Comment:
I like the additional help command that you have included 👍

Reply:
thank you!